lib/gcra/redis_store.rb

```diff
@@ -13,19 +15,23 @@ class RedisStore
   return 1
   EOF
     CAS_SCRIPT_MISSING_KEY_RESPONSE = 'key does not exist'.freeze
+    SCRIPT_NOT_IN_CACHE_RESPOSNE = 'NOSCRIPT No matching script. Please use EVAL.'
 
     def initialize(redis, key_prefix)
       @redis = redis
       @key_prefix = key_prefix
+      @cas_sha = Digest::SHA1.hexdigest(CAS_SCRIPT)
```

Comment:
Since the script does not change during the lifetime of the client, do we actually need to calculate this in the initializer?

Is there any reason which I am not seeing why you decided against calculating that on load?
e.g.
```ruby
CAS_SHA = Digest::SHA1.hexdigest(CAS_SCRIPT).freeze
```
at the top

Reply:
I have a predisposition against doing too much work during ruby's parse-phase.  In my opinion, requiring a file should be as quick as possible and if part of requiring that file is generating a SHA, that adds extra overhead and makes `require` slightly slower.

What about precomputing the hexdigest and adding a spec that asserts it is correct?
```
CAS_SHA = "89118e702230c0d65969c5fc557a6e942a2f4d31".freeze
```

Reply:
Yeah, it makes sense to not do to much work on require.
The way we are using it, it does not matter whether it happens on require or in the initializer (generally just one instance of `GCRA::Ruby` per code instance)

Your suggestion of adding it specifically and ensuring that it matches in a spec would get around both cases and we'd just have the computation once when changing the script 👍 
I like that approach

Reply:
Great!  Done in 0828b0f9cb3c7f7484c0e1e98a0372577fcb3efe.

Reply:
Very nice :) thank you